packages/shared/src/utils/constants/contract.tsx

```diff
@@ -12,7 +12,18 @@ export const getLocalizedBasis = () => ({
  * components can be undef or an array containing any of: 'start_date', 'barrier', 'last_digit'
```

Comment:
You don't have to review this file! I just merged it with my branch to fix TS errors for helpers